app/helpers/shop_mail_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ShopMailHelper
+  def mail_long_datetime_format
```

Comment:
an example of a date that follows the format below as method doc will come in handy

Reply:
Thanks, @sauloperez. Done here: fc033822984cbd3ab183fe69874540f0f47ee9c9

Reply:
Eep! :grin: After reading again, it's quite obvious that's what @sauloperez meant, and it makes a lot more sense. Sorry for the trouble, and thanks for checking! I pushed the change: ad8b120e4869c53a0ec04abff8bdd63de4191d7c PR checks remain green.